qa-tests-backend/src/main/groovy/services/SignatureIntegrationService.groovy

```diff
@@ -0,0 +1,75 @@
+package services
+
+import io.grpc.Status
+import io.grpc.StatusRuntimeException
+import io.stackrox.proto.api.v1.EmptyOuterClass
+import io.stackrox.proto.api.v1.SignatureIntegrationServiceGrpc
+import io.stackrox.proto.api.v1.SignatureIntegrationServiceOuterClass
+import io.stackrox.proto.storage.ImageIntegrationOuterClass
+import io.stackrox.proto.storage.SignatureIntegrationOuterClass
+import util.Timer
+
+class SignatureIntegrationService extends BaseService {
+    static getSignatureIntegrationClient() {
+        return SignatureIntegrationServiceGrpc.newBlockingStub(getChannel())
+    }
+
+    static createSignatureIntegration(SignatureIntegrationOuterClass.SignatureIntegration integration) {
+        SignatureIntegrationOuterClass.SignatureIntegration createdIntegration
+        Timer t = new Timer(15, 3)
+        while (t.IsValid()) {
+            try {
+                createdIntegration = getSignatureIntegrationClient().postSignatureIntegration(integration)
+                println "Signature integration created: ${createdIntegration.getName()}: ${createdIntegration.getId()}"
+                break
+            } catch (Exception e) {
+                println "Unable to create signature integration ${integration.getName()}: ${e.getMessage()}"
+            }
+        }
+
+        if (!createdIntegration || !createdIntegration.getId()) {
+            println "Unable to create signature integration"
+            return ""
+        }
+
+        SignatureIntegrationOuterClass.SignatureIntegration foundIntegration
+        t = new Timer(15, 3)
+        while (t.IsValid()) {
+            try {
+                foundIntegration =
+                        getSignatureIntegrationClient().getSignatureIntegration(
+                                getResourceByID(createdIntegration.getId()))
+                if (foundIntegration) {
+                    println "Integration found after creation: " +
```

Comment:
Do we need logs for success code paths?

Reply:
Not sure, I looked at `ImageIntegrationService` as sample. I can also remove it if you feel strongly about it, don't have a preference on it tbh.

Reply:
Sounds good to me. I don't see a reason to keep them - they potentially will pollute logs and the reason people look at the logs is mostly to see out-of-regular behaviour.